client/runtime/src/main/java/io/quarkiverse/openapi/generator/OpenApiGeneratorConfig.java

```diff
@@ -3,18 +3,20 @@
 import java.util.Map;
 import java.util.Optional;
 
-import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithParentName;
 import io.smallrye.config.common.utils.StringUtil;
 
 /**
  * This class represents the runtime configurations for the openapi-generator extension.
  */
-@ConfigRoot(name = OpenApiGeneratorConfig.RUNTIME_TIME_CONFIG_PREFIX, phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
-public class OpenApiGeneratorConfig {
+@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
```

Comment:
@mcruzdev still build time. Can you plese wait until #943 is merged so we can rebase this ASAP?

Reply:
Perfect, it makes sense!